### DISCUSSION TITLE
Can `StringFormat` support `<placeholder>` or `[placeholder]`?

### DISCUSSION THREAD

**Discussion by xingyutangyuan** [google/mug#65] [Ideas]
StringFormat's curly brace syntax is cool. But can it support other placeholder syntax like `<placeholder>` or `${placeholder}`?

**Comment by fluentfuture** [comment]
Internally we have a variant that supports using square brackets.

Adding the support is easy, but it complicates the ErrorProne check a little.

We haven't seen high demand for adding alternative placeholders yet.

Maybe you can describe why they are important to you?
